pkg/controllers/flannelmigration/migration_controller.go

```diff
@@ -424,21 +427,23 @@ func (c *flannelMigrationController) runIpamMigrationForNodes() ([]*v1.Node, err
 		}
 	}
 
-	// Now we have a list of nodes which does not include master node and controllerNode.
+	// Now we have a list of nodes which does not include control plane nodes, nor any node running this controller.
 	// We need to sort the list so that, if controller failed and restarted, it will start
 	// to process the same node again. This is to prevent migration controller to migrate another
 	// node without addressing previous failure.
 	sort.SliceStable(nodes, func(i, j int) bool { return nodes[i].Name < nodes[j].Name })
 
-	if masterNode != nil {
-		log.Infof("Master node %s is last node to be migrated.", masterNode.Name)
-		nodes = append(nodes, masterNode)
+	if len(masterNodes) != 0 {
+		log.Infof("Adding control plane nodes to end of queue.", masterNodes)
```

Comment:
_go vet_ complained that this format string is missing a directive. Perhaps:
```suggestion
		log.Infof("Adding control plane nodes to end of queue: %v.", masterNodes)
```